src/components/AudioErrorBanner/AudioErrorBanner.module.css

```
@@ -21,6 +21,24 @@
+   BREAKPOINTS.mobileMax). The banner renders at the app fragment root, a
+   sibling of MobileShell, so it has no [data-layout-tier] ancestor to scope on
+   — a width media query is the only reliable hook. Lift it above the always-
+   open bottom-sheet peek (--mobile-sheet-peek, defined in semantic.css ←
+   --token-mobile-sheet-peek; the same token drives MobileShell.module.css
+   .stage padding-bottom) and inside the bottom safe
+   area so it clears the home indicator. z-index already sits above the sheet
+   (--token-z-overlay-drawer 201 > --token-z-mobile-sheet 100). */
+@media (max-width: 767px) {
+  .banner {
+    bottom: calc(
+      var(--mobile-sheet-peek, 96px) + env(safe-area-inset-bottom, 0px) +
```

Stylelint (17.12.0)

[error] Unexpected newline after "+" (scss/operator-no-newline-after)

(scss/operator-no-newline-after)